ci/docker/doc.build.Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG DOCKER_IMAGE_BASE_BUILD=cr.ray.io/rayproject/oss-ci-base_build
+ARG DOCKER_IMAGE_BASE_BUILD=cr.ray.io/rayproject/oss-ci-base_build-py3.12
```

Comment:
where is this built btw?

Reply:
it's from here https://github.com/ray-project/ray/blob/master/.buildkite/base.rayci.yml#L26